packages/react-adminer/src/components/Edit.tsx

```diff
@@ -197,32 +213,30 @@ const EditChild: React.FC<Props> = ({ entityConfig, entityName, id }) => {
 						<Alert message="You are creating new entity" type="info" showIcon />
 					</Box>
 				)}
-				<SectionBox
-					primitiveFields={primitiveFields}
-					isNewForm={isNewForm}
-					onChange={onChange}
-					getValidDateValue={getValidDateValue}
-					state={state}
-					selectedSection="primarySection"
-				/>
-				<Divider />
-				<SectionBox
-					primitiveFields={primitiveFields}
-					isNewForm={isNewForm}
-					onChange={onChange}
-					getValidDateValue={getValidDateValue}
-					state={state}
-					selectedSection="secondarySection"
-				/>
-				<Divider />
-				<SectionBox
-					primitiveFields={primitiveFields}
-					isNewForm={isNewForm}
-					onChange={onChange}
-					getValidDateValue={getValidDateValue}
-					state={state}
-				/>
-				<Divider />
+				<Space direction="vertical" size="large" style={{ display: 'flex' }}>
+					<SectionBox
+						fields={sectionFields(primitiveFields, 'primarySection')}
```

Comment:
co je `primarySection` a co je `secondarySection`? Jak to bude fungovat když si udělám sekci `fakturační údaje`?

Neměl by ten kód fungovat spíš tak že ti ty sekce vytahá sám? Nespoléháš na to že tam budou právě dvě a to `primarySection` a  `secondarysection`?